### ISSUE TITLE
Install as dev dependency

### ISSUE THREAD

**Issue by Maxpain177** [adelsz/pgtyped#106]
Our docker images have only production node_modules, without dev ones.
When I try to use `@pgtyped/cli` as dev dependency I get runtime error `Error: Cannot find module 'chalk'`. I tried to install `@pgtyped/cli` as a non-dev dependency and got error `Error: Cannot find module 'typescript'`

**Comment by adelsz** [comment]
Can you please try again with v0.7.3?
Typescript is now an explicit peer dependency.

**Comment by Maxpain177** [comment]
@adelsz 
```
maxpain@Maxpains-Pro api % node .build/index.js
internal/modules/cjs/loader.js:1032
  throw err;
  ^

Error: Cannot find module 'typescript'
Require stack:
- /Users/maxpain/dev/gameflare/backend/api/node_modules/@pgtyped/query/lib/loader/typescript/index.js
- /Users/maxpain/dev/gameflare/backend/api/node_modules/@pgtyped/query/lib/index.js
- /Users/maxpain/dev/gameflare/backend/api/.build/utils/graphql/context.js
- /Users/maxpain/dev/gameflare/backend/api/.build/index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:1029:15)
    at Function.Module._load (internal/modules/cjs/loader.js:898:27)
    at Module.require (internal/modules/cjs/loader.js:1089:19)
    at require (internal/modules/cjs/helpers.js:73:18)
    at Object.<anonymous> (/Users/maxpain/dev/gameflare/backend/api/node_modules/@pgtyped/query/lib/loader/typescript/index.js:6:38)
    at Module._compile (internal/modules/cjs/loader.js:1200:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1220:10)
    at Module.load (internal/modules/cjs/loader.js:1049:32)
    at Function.Module._load (internal/modules/cjs/loader.js:937:14)
    at Module.require (internal/modules/cjs/loader.js:1089:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/maxpain/dev/gameflare/backend/api/node_modules/@pgtyped/query/lib/loader/typescript/index.js',
    '/Users/maxpain/dev/gameflare/backend/api/node_modules/@pgtyped/query/lib/index.js',
    '/Users/maxpain/dev/gameflare/backend/api/.build/utils/graphql/context.js',
    '/Users/maxpain/dev/gameflare/backend/api/.build/index.js'
  ]
}
```

**Comment by adelsz** [comment]
Did you install `typescript`?
It is a _peer_ dependency that isn't installed as part of pgtyped.

**Comment by adelsz** [comment]
Yes, TS is a prod peer dependency atm.
This is planned to be fixed in #91.
